### DISCUSSION TITLE
Failed to parse channel schema on /tf no such type: foxglove.FrameTransform

### DISCUSSION THREAD

**Discussion by kebiqiuyw** [foxglove/community#1036] [Q&A]
foxglove.FrameTransform is an embedded protobuf type， I can replay the demo mcap which include foxglove.FrameTransform messages buf when i add the foxglove.FrameTransform channel, an error happens， it hints no such type on version 2.12.1. How to fix it, anyone can help me?

**Comment by jtbandes** [comment]
@kebiqiuyw It looks like you closed this discussion; does that mean you were able to find a solution?

**Comment by kebiqiuyw** [comment]
Yes. I made a misstake in my demo code when create the channel.&nbsp;&nbsp;


I would like to ask about the paid version：
1. Are there technical support representatives available in China？
2. In what ways&nbsp; the&nbsp; tech support&nbsp; will provide？（only email？）Because i wanna do some customized requests.


Thanks！





---Original---
From: "Jacob ***@***.***&gt;
Date: Tue, Sep 10, 2024 00:19 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [foxglove/community] Failed to parse channel schema on /tf nosuch type: foxglove.FrameTransform (Discussion #1036)




 
@kebiqiuyw It looks like you closed this discussion; does that mean you were able to find a solution?
